model-archiver/model_archiver/arg_parser.py

```diff
@@ -19,101 +18,130 @@ class ArgParser(object):
     @staticmethod
     def export_model_args_parser():
```

Comment:
model archiver needs to be extended to support new runtime types: eg. onnx, tensorRT